### DISCUSSION TITLE
Is there a more universal frontend solution that can be directly integrated?

### DISCUSSION THREAD

**Discussion by imac8t** [ayamir/nvimdots#1359] [General]
感觉自定义配置这方面还是不太详细呢。

**Comment by charliie-dev** [comment]
https://github.com/ayamir/nvimdots/wiki

**Reply by imac8t** [reply]
觉得不够详细。。。

**Reply by Jint-lzxy** [reply]
Hey imac8t! Can u share more about why u think it's not detailed enough? If I understand u correctly neovim doesn't have a _generic_ integration like VSCode's marketplace. The plugins here are pretty general, just like using VSCode without any add-ons. Instead of a simple click to set up for front-end dev, _you_ gotta tweak things to fit ur style. For example, u can set up LSPs for the languages u use or grab extra plugins that match ur workflow. We do have better support for some languages but that's simply bc the maintainers know them well. If u feel a bit lost, check out some [intro](https://youtu.be/m8C0Cq9Uv9o) [vids](https://youtu.be/w7i4amO_zaE) on Youtube or try [kickstart.nvim](https://github.com/nvim-lua/kickstart.nvim) to help u get started!
